app/file.go

```diff
@@ -673,6 +673,20 @@ func (t *uploadFileTask) runPlugins() *model.AppError {
 }
 
 func (t *uploadFileTask) preprocessImage() *model.AppError {
+	// If SVG, attempt to extract dimensions and then return
+	if t.fileinfo.MimeType == "image/svg+xml" {
```

Comment:
@levb I'm not very familiar with the new file upload path, but why is this being done here and not in `model.GetFileInfoForBytes`? I see that this streams the data, but then I'm not sure why the old function wasn't replaced entirely.

Reply:
Hmmm, it's a little complicated. 

The new UploadFile code path does not use `model.GetInfoForFilename` - primarily because it was not streaming-friendly, but I also questioned parsing images in the model package. My personal expectation is that `model` does not perform lengthy nor complex operations.

However, there are 2 places where it is still used in the legacy DoUploadFile implementation which I'd like to remove; and indirectly from https://github.com/mattermost/mattermost-server/blob/8106a2ba6676577bc96aace2a200e4cc0c27a4c4/app/file.go#L222. 

Once I get https://github.com/mattermost/mattermost-server/pull/10056 merged, I'd love to refactor all of the prior file upoload/fileinfo extraction code out.